app/uk/gov/hmrc/bindingtariffclassification/model/Application.scala

```diff
@@ -27,10 +27,11 @@ sealed trait Application {
 
 case class BTIApplication
 (
-  offline: Boolean = false,
   holder: EORIDetails,
   contact: Contact,
   agent: Option[EORIDetails] = None,
+  offline: Boolean = false,
+  itemsCategory: String,
```

Comment:
should it not be itemCategory? rather than item**s**Category

Reply:
I am not sure. 
We should ask our BAs.

But I think that with a BTI application you can trade multiple items, as long as they belong to the same category.
For example, a trader could sell tablets, smartphones, phones, etc.
All these items belong to the same category.
So, I feel the term `itemsCategory` is more appropriate.